### PR TITLE
Microverse project

### PR DESCRIPTION
**Pull request by TEMESGEN-G25**
In this simple project I:
Added an index.html file.
Added an index.css file.
Added a test.md file.
Set up a linter for HTML & CSS.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
Comment (elmar8287) [inline .gitignore]: - [ ] Please, add into .gitignore file the test.md, as requered by the project.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# github-flow
\ No newline at end of file
+This is my first time submiting a project to Microverse. There wasn't any complicated code nevertheless I enjoyed working at this simple project. I created a .gitignore file. I set up a linter for HTML & CSS. I added 3 files an HTML, CSS & a test.md. The test.md file is the only file that's added to the .gitignore file.
```
Comment (elmar8287) [inline README.md]: - [ ] Please, create a detailed README file. Use [MICROVERSE template](https://github.com/microverseinc/readme-template), as required.

File: index.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.header{
+    color: brown;
+}
```
Comment (elmar8287) [inline index.css]: - [ ] Please, fix the linter issue
Screen Shot 2022-01-06 at 11 06 48

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @NatiG25 ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [ ] Please, consider comments under review
- [ ] Please, consider that the PR (Pull Request) should have a descriptive title telling you what changes are inside. 
- [ ] Please, consider that the PR should have a short summary describing introduced changes in more detail. You have not provided a description.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @NatiG25, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- Please write a meaningful title for your pull request and add a short summary of changes introduced in the pull request as requested by the previous reviewer. Please check this [LINK](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for help.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @NatiG25, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**